<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>79</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QToolButton" name="btnUp">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="btnWidget" native="true">
     <layout class="QVBoxLayout" name="btnLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnDown">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
